docs/src/dev/provider/index.asciidoc

```diff
@@ -1007,56 +1167,41 @@ dependent on the operation being performed.  A `RequestMessage` has these fields
 [width="100%",cols="3,10",options="header"]
 |=========================================================
 |Key |Description
-|requestId |A link:http://en.wikipedia.org/wiki/Globally_unique_identifier[UUID] representing the unique identification for the request.
-|op |The name of the "operation" to execute based on the available `OpProcessor` configured in the Gremlin Server.  To evaluate a script, use `eval`.
-|processor |The name of the `OpProcessor` to utilize. The default `OpProcessor` for evaluating scripts is unnamed and therefore script evaluation purposes, this value can be an empty string.
-|args |A `Map` of arbitrary parameters to pass to Gremlin Server.  The requirements for the contents of this `Map` are dependent on the `op` selected.
+|gremlin |The script to be executed by the server.
```

Comment:
is this table a duplicate of what was in the HTTP section? any reason to have it in both places? shouldn't this just link to the HTTP section of the table? also, note this version of the table still references the "op" code.